module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,65 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+#include <utility>
+#include <vector>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(std::array<std::array<uint8_t, width>, height>& bitmap) {
```

Comment:
enter before arguments will make this code more readable, another solution is using aliases for long types

Reply:
Aplied aliasing. Very thanks for suggestion :)